### PR TITLE
foreman maintain to satellite maintain

### PR DESCRIPTION
**Pull request by tstrych**
we should use downstream name

it is just symlink
```
ls -l /usr/bin/satellite-maintain
....  /usr/bin/satellite-maintain -> /usr/bin/foreman-maintain
```

### CODE REVIEW

File: testfm/content.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
 
 class Content(Base):
-    """Manipulates Foreman-maintain's content command"""
+    """Manipulates Satellite-maintain's content command"""
```
Comment (Gauravtalreja1) [inline testfm/content.py:23]: can you check and replace f-m to s-m in class methods as well?

File: tests/test_service.py [hunk 2]
```diff
@@ -179,7 +179,7 @@ def test_positive_automate_bz1626651(ansible_module):
 
 @pytest.mark.capsule
 def test_positive_service_status_clocale(ansible_module):
-    """Foreman-maintain service on C locale
+    """Satellite-maintain service on C locale
```
Comment (Gauravtalreja1) [inline tests/test_service.py:182]: Can you check and update tests docstrings as well for this change?

**Review by Gauravtalreja1 — CHANGES_REQUESTED** [review]

**Comment by Gauravtalreja1** [comment]
@tstrych any status update?

File: testfm/content.py (hunk 1 above)
Comment (tstrych) [reply testfm/content.py:23]: still required: 
`testfm` ->  `testsm` (all imports in all files, repo name, ...)
`fm` -> `sm`

**Comment by tstrych** [comment]
@Gauravtalreja1 PR has been updated.

File: testfm/content.py (hunk 1 above)
Comment (shubhamsg199) [reply testfm/content.py:23]: I agree with @Gauravtalreja1 here, changing all imports in all files, repo name will result in numerous changes and would suggest proceeding with changes made for now.

Comment (tstrych) [reply testfm/content.py:23]: @Gauravtalreja1 so it's upon you whether you want to merge this.

File: tests/test_maintenance_mode.py [hunk 3]
```diff
@@ -5,14 +5,14 @@
...
+    """Test satellite-maintain maintenance-mode subcommand
 
     :id: 51d76219-d3cc-43c0-9894-7bcb75c163c3
 
     :setup:
-        1. foreman-maintain should be installed.
+        1. satellite-maintain should be installed.
         2. active sync-plans
-        3. set :manage_crond: true in /etc/foreman-maintain/foreman_maintain.yml'
+        3. set :manage_crond: true in /etc/satellite-maintain/foreman_maintain.yml'
```
Comment (Gauravtalreja1) [inline tests/test_maintenance_mode.py:15]: @tstrych sorry for being nitpicky here, this is a config file, this shouldn't be changed, can you update this?
```suggestion
        3. set :manage_crond: true in /etc/foreman-maintain/foreman_maintain.yml'
```

File: tests/test_packages.py [hunk 4]
```diff
@@ -142,16 +142,16 @@ def test_positive_fm_packages_install(ansible_module, setup_packages_lock_tests)
...
+        1. satellite-maintain should be installed.
 
     :steps:
         1. Run satellite-installer --lock-package-versions
-        2. Run foreman-maintain packages status
-        3. Run foreman-maintain packages is-locked
-        4. Try to install/update package using FM packages install/update command.
+        2. Run satellite-maintain packages status
+        3. Run satellite-maintain packages is-locked
+        4. Try to install/update package using SM packages install/update command.
```
Comment (Gauravtalreja1) [inline tests/test_packages.py:151]:
```suggestion
        4. Try to install/update package using satellite-maintain packages install/update command.
```

File: tests/test_packages.py [hunk 5]
```diff
@@ -164,7 +164,7 @@ def test_positive_fm_packages_install(ansible_module, setup_packages_lock_tests)
     for result in contacted.values():
         assert result["rc"] == 1
         assert "Use foreman-maintain packages install/update <package>" in result["stdout"]
-    # Test whether FM packages install/ update command works as expected.
+    # Test whether SM packages install/ update command works as expected.
```
Comment (Gauravtalreja1) [inline tests/test_packages.py:167]:
```suggestion
    # Test whether satellite-maintain packages install/ update command works as expected.
```

File: tests/test_upgrade.py [hunk 6]
```diff
@@ -101,22 +101,22 @@ def test_positive_repositories_validate(ansible_module):
...
-        1. Run foreman-maintain upgrade check/run command.
-        2. Run foreman-maintain upgrade check/run command with disable-self-upgrade option.
+        1. Run satellite-maintain upgrade check/run command.
+        2. Run satellite-maintain upgrade check/run command with disable-self-upgrade option.
 
     :BZ: 1649329
 
     :expectedresults:
-        1. It updates FM to latest version and gives message to re-run command.
+        1. It updates SM to latest version and gives message to re-run command.
```
Comment (Gauravtalreja1) [inline tests/test_upgrade.py:119]:
```suggestion
        1. Update satellite-maintain package to the latest version and ask a user to re-run the command.
```

**Review by Gauravtalreja1 — COMMENTED** [review]
@tstrych I appreciate you extending this to TestFM for updating the renaming of docs, CLIs, tests, and constants. Thank you!
Please consider few more suggested changes

File: tests/test_maintenance_mode.py (hunk 3 above)
Comment (tstrych) [reply tests/test_maintenance_mode.py:15]: no you are correct here :) I tried to not change paths, I missed this one though

**Review by Gauravtalreja1 — APPROVED** [review]
@tstrych ACK, Thanks for the changes.

**Review by jameerpathan111 — APPROVED** [review]

**Pull request merged**